src/plugin/cursus/Resources/translations/cursus.en.json

```diff
@@ -97,5 +97,7 @@
     "session_info": "Session information",
     "download_presence": "Download the certificate of presence (.pdf)",
     "show_catalog": "Show training catalog",
-    "registration_auto": "All session participants are automatically registered for this event."
+    "registration_auto": "All session participants are automatically registered for this event.",
+    "session_days": "Days",
+    "session_hours": "Hours"
```

Comment:
IMO it already exist in `platform` domain.

Reply:
Yes, a translation already exit in ```platform``` but only for *unit* but not for *label*. Do you want I use the translation already exist ?

Reply:
You will no longer need it. The label for your new field of type `duration` will be `Duration`. And your new type will be similar to the `time` type which uses the unit translations between inputs.

Also note, we may merge `time` and `duration` later.